inc/tabbed-pages.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * Meta Box for adding tab navigation to a page
+ *
+ * @package   Nightingale-2-0
+ * @copyright NHS Leadership Academy, Very Twisty
+ * @version   Feburary 2020
```

Comment:
Typo in "February"

Reply:
Also, is "February 2020" a version? The correct version?

Reply:
:good spot!